pkg/web/src/modules/editClusterSlice.ts

```diff
@@ -61,12 +68,12 @@ const slice = createSlice({
     updateCluster: (state, action: PayloadAction<{ id: string; data: Partial<Omit<Cluster, 'id'>> }>) => {
       // Remove /
       // craneUrl = http://localhost:3000/ => http://localhost:3000
-      const { craneUrl } = action.payload.data;
-      if (craneUrl) {
-        action.payload.data.craneUrl = craneUrl.endsWith('/')
-          ? craneUrl.substring(0, craneUrl.lastIndexOf('/'))
-          : craneUrl;
-      }
+      // const { craneUrl } = action.payload.data;
```

Comment:
Not the correct way to implement it ,maybe try chatGPT ?

Reply:
If a backslash appears at the end of the URL, it needs to be removed.
If not the app will be displayed an error.

Please don't change it.

Reply:
Do you use this regex for validation? The problem here is' substring(0, craneUrl.lastIndexOf('/')) '